icu4c/source/test/intltest/numbertest_simple.cpp

```diff
@@ -0,0 +1,46 @@
+// © 2018 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_FORMATTING
+
+#include "numbertest.h"
+#include "unicode/numberformatter.h"
+
+#include <cmath>
+#include <numparse_affixes.h>
```

Comment:
Please use double quotes for including ICU headers.

Reply:
oops. I think VSCode added this automatically.